roles/webserver/tasks/main.yml

```
@@ -7,7 +7,7 @@
 - name: Create crontab entries for scheduled job
   cron:
     name: send_recent_posts_to_online_members
-    user: timeoverflow
+    user: {{ app_user }}
```

Does it work? I would expect ansible complainint about quoting.
Not in templates, but in task files and other ansible-yaml files, jinja templates `{{ ... }}` as far as I've seen, must be surrounded by double quotes. Like:
```suggestion
    user: "{{ app_user }}"
```